data/json/mapgen/nested/cave_nested.json

```diff
@@ -0,0 +1,394 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "nested_mapgen_id": "cave_nw_1",
+    "object": {
+      "rotation": [ 0, 0 ],
+      "mapgensize": [ 11, 11 ],
+      "rows": [
+        "...........",
+        "     ...  .",
+        "     ...  .",
+        "...   ..  .",
+        "....      .",
+        ".....    ..",
+        ".....     .",
+        ".....     .",
+        ".....     .",
+        ".....      ",
+        ".....      "
+      ],
+      "terrain": { ".": "t_rock", " ": "t_rock_floor" },
```

Comment:
These have the same definition across all 16 of these nested parts--you could just make a small palette and reference it in all of them.

Reply:
I considered this variant, I just thought that total number of lines will be more than current situation: 1 line `"palettes": [ "foo" ]` plus 1 line `"terrain": { ".": "t_rock", " ": "t_rock_floor" }` in palette definition.

Reply:
Palettes can include monsters as well, so you'd actually move all of this:

```
"terrain": { ".": "t_rock", " ": "t_rock_floor" },
"monster": { " ": { "monster": "mon_sewer_rat", "chance": 20 } }
```
in exchange for the single palette reference. 

It'd be approximately half the number of lines overall, as well as ensuring consistency (which would be the primary goal). It'd be a different story if these all had different values, but they don't (and if there are a few exceptions they can always override the palette locally, as the inline terrain/monster definitions take precedence over the ones in the palette.

Reply:
Okay, used the palette.